src/arch/xtensa/include/arch/lib/cpu.h

```diff
@@ -28,6 +28,8 @@ int arch_cpu_enabled_cores(void);
 
 int arch_cpu_restore_secondary_cores(void);
 
+int arch_cpu_writeback_secondary_cores(void);
+
```

Comment:
to commit description: sorry, I don't see where in `platform_pg_int_handler()` all secondary cores are disabled?

Reply:
pm_runtime_put